R/pathwayEnrich.R

```diff
@@ -0,0 +1,131 @@
+
+# pathwayEnrich.R
+#'
+#' \code{PathwayEnrichment} a function that perform a pathway enrichment analysis 
+#' Return a barplot that show the enriched pathway and its p value(fisher test) that is smaller than 
+#' the ajusted p value. The null hypothesis for each fisher test: the pathway
+#' is not enriched in the given gene set.
+#'
+#' @param geneSet A list of gene symbol in the system that you want to perform pathway
+#'  enrichment analysis.
+#'  
+#' @param method method of cutoff that the visualization plot method can be BH or 
+#' bonferroni.
+#' reactInfo
+#' @param reactInfo a dataframe map the reactome pathway id to the reference gene symbol 
+#' 
+#' @return a barplot that show the enriched pathway and its p value(fisher test) 
+#' that is smaller than the ajusted p value.
+#' 
+#' @import ggplot2
+#' @importFrom stats fisher.test
+#' @importFrom stats p.adjust
+#' @author {Yufei Yang} (aut)
+#' @examples
+#' # Picking sample gene list to find the pathway enrichment and p value
+#' # Call the symComp helper function to generate list
+#' sampleGeneList<-c("CREBBP","NR1H3","RORA","SREBF1","SMARCD3")
+#' reactInfo<- fetchData("ReactomeSym")
```

Comment:
Is this ever not with the ReactomeSym data? If not you could consider just leaving that parameter out and running fetchData() inside the function

Reply:
ReactomeSym is newly added data from fetchData()

Reply:
sample gene list and fetchData() are both needed reactomeSym just mapping the Reactome id to the gene symbol.